external/include/netutils/netlib.h

```diff
@@ -146,6 +146,11 @@ uint8_t netlib_ipv6netmask2prefix(FAR const uint16_t *mask);
 void netlib_prefix2ipv6netmask(uint8_t preflen, FAR struct in6_addr *netmask);
 #endif
 
+/* Network Monitor */
+
+#ifdef CONFIG_NET_NETMON
+int netlib_netmon(FAR const char *command);
```

Comment:
please change the function definition. it's not good to have parameters in API  as commands.

Reply:
The function name and input arguments have changed, but not reflected in the header file. Sorry for my mistake. Modification completed.